src/interfaces/app-details.ts

```diff
@@ -1,5 +1,6 @@
-export type RecordVisibility = 'member' | 'item';
-
-export interface AppIdentification {
-  app: string;
+export enum AppDataVisibility {
+  ITEM = 'item',
+  MEMBER = 'member',
 }
+
+export type RecordVisibility = 'member' | 'item' | AppDataVisibility;
```

Comment:
Why does this type expose the same values as `AppDataVisibility` but without the typed enum?